weedcoco/index/thumbnailing.py

```diff
@@ -0,0 +1,27 @@
+import argparse
+import os
+import pathlib
+from PIL import Image
+
+
+def thumbnailing(thumbnails_dir, repository_dir, THUMBNAIL_SIZE=(300, 300)):
+    for dirpath, dirnames, filenames in os.walk(repository_dir):
+        for filename in filenames:
+            if filename.lower().endswith((".png", ".jpg", ".jpeg", ".tiff")):
```

Comment:
should we only be processing files that do not already exist in the thumbnails dir?